hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/PreHandleDispatcher.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.spi;
+
+import com.hederahashgraph.api.proto.java.TransactionBody;
+
+public interface PreHandleDispatcher {
```

Comment:
I have had doubts as to whether we really need this in the SPI. Maybe we will, but I'm not sure. Maybe we can start with it being an implementation detail in the `hedera-app` module?

Reply:
Makes sense

Reply:
Moved it to the `hedera-app` module